### PR TITLE
Adding Py Dep

### PR DESCRIPTION
**Pull request by psteyer**
Adding Py dependency see following Jira ticket for more information:
https://anaconda.atlassian.net/browse/CVE-566

## changes
- adding `py`
- bumping build #

### CODE REVIEW

**Review by ryanskeith — APPROVED** [review]
LGTM.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ requirements:
     - python
     - pytest >=3.8
     - py-cpuinfo
+    - py
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:29]:
```suggestion
```
It doesn't rely on `py` anymore. See:
- https://github.com/ionelmc/pytest-benchmark/issues/226
- https://github.com/ionelmc/pytest-benchmark/commit/c2e860f628116473e3134aba21492c04650d00a0
- https://github.com/ionelmc/pytest-benchmark/blob/v4.0.0/setup.py#L69-L71
- https://github.com/ionelmc/pytest-benchmark/blob/7ef2645ae76dfd7a5928ae552fc7f5b955616074/src/pytest_benchmark/cli.py#L4-L5

Comment (psteyer) [reply recipe/meta.yaml:29]: Here they seem to still be requiring it.

- https://github.com/ionelmc/pytest-benchmark/blob/728752d2976ef53fde7e40beb3e55f09cf4d4736/src/pytest_benchmark/csv.py#L6
- https://github.com/ionelmc/pytest-benchmark/blob/728752d2976ef53fde7e40beb3e55f09cf4d4736/src/pytest_benchmark/histogram.py#L3

Comment (skupr-anaconda) [reply recipe/meta.yaml:29]: They rely on `py.path.local` only so there can be a way to patch the package to use **pytest's** `py` iplementetion

Comment (psteyer) [reply recipe/meta.yaml:29]: Would it be safer to keep it in now, remove the `pytest` hotfix, and remove it in the future?

Comment (skupr-anaconda) [reply recipe/meta.yaml:29]: Yes.

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**